### PR TITLE
Traditional chinese

### PR DESCRIPTION
**Pull request by kairstenfay**
Add traditional Chinese (zh-Hant) report translation.
Update external links to use translations where available.

### CODE REVIEW

**Comment by kairstenfay** [comment]
[New language: Traditional Chinese](https://trello.com/c/x10272OV/552-new-language-traditional-chinese)

**Comment by kairstenfay** [comment]
["never tested" for all languages](https://trello.com/c/WnW5D614/555-never-tested-for-all-languages)

**Review by tsibley — APPROVED** [review]
LGTM assuming our language reviewers give the thumbs up. The updates to existing templates will require us to manually trigger a whole new PDF regen on the backoffice, yeah?

**Review by joverlee521 — CHANGES_REQUESTED** [review]
I would definitely recommend increasing the font size for traditional Chinese. Even I have squint to read it with the current font size.

**Comment by kairstenfay** [comment]
> LGTM assuming our language reviewers give the thumbs up. The updates to existing templates will require us to manually trigger a whole new PDF regen on the backoffice, yeah?

We're pending review now. 

Yes, someone will have to manually create new PDFs for zh-Hant. This task is tracked in the Trello card.

**Comment by kairstenfay** [comment]
> I would definitely recommend increasing the font size for traditional Chinese. Even I have squint to read it with the current font size.

Thank you for this feedback!

**Comment by kairstenfay** [comment]
`zh-Hant` now has a working bold-face font (thanks for catching this, @joverlee521 !). Unfortunately this required the download of Noto Sans CJK fonts which, on my machine, slows down the Docker image generation by about 2 minutes.

**Comment by kairstenfay** [comment]
I just force-pushed after rewording a commit. No code changes.

**Comment by kairstenfay** [comment]
Just pushed a new commit with downloaded Chinese fonts in the `fonts` directory. The Docker image is no longer burdened by downloading the huge Noto CJK package.

**Comment by kairstenfay** [comment]
I just force-pushed a reword and a minor whitespace edit. I don't intend to make further changes at this time.

File: scan/report-zh-Hans.tex [hunk 1]
```diff
@@ -28,8 +28,9 @@
 \setmonofont{DroidSansMono}[
   Extension   = .ttf,
   UprightFont = * ]
-\setCJKmainfont{NotoSansCJK}[
-  Extension   = .ttc,
+\setCJKmainfont{NotoSansCJKsc}[
+  Path        = /output/fonts/,
```
Comment (tsibley) [inline scan/report-zh-Hans.tex:32]: Hardcoding this Path means there's now a hard requirement on a `/output` mount and no chance of this working outside of Docker. I think instead the font files could be copied into the Docker image and referred to by a relative path in these templates.

Comment (tsibley) [reply scan/report-zh-Hans.tex:32]: Thinking about this a bit more: we're using Noto Sans CJK now for Simplified Chinese, Traditional Chinese, and in the future (presumably) Korean. That just leaves Japanese as the only language which is in the larger `fonts-noto-cjk` package but that we won't use, so maybe the returns are dwindling.

I think the only remaining win might be if individual PDF generation (not Docker image generation) is faster when TeX only has to load the single-language font files. Not sure about this.

**Comment by kairstenfay** [comment]
Rebased onto master.
The zh-Hant report has been verified by a native speaker.

**Review by tsibley — COMMENTED** [review]
Somehow I never submitted the comments below. Just found them because discussion in Slack triggered my memory. Submitting now for posterity, even though this PR is merged already.

**Pull request merged**